esp-hal/src/clock/clocks_ll/esp32h2.rs

```diff
@@ -239,54 +234,49 @@ fn clk_ll_bus_update() {
 }
 
 fn regi2c_enable_block(block: u8) {
-    reg_set_bit(MODEM_LPCON_CLK_CONF_REG, MODEM_LPCON_CLK_I2C_MST_EN);
-    reg_set_bit(I2C_MST_DATE_REG, I2C_MST_CLK_EN);
+    let modem_lpcon = unsafe { &*crate::peripherals::MODEM_LPCON::ptr() };
 
-    // Make I2C_MST_ANA_CONF2 in I2C_MST_ANA_CONF2_REG be 0
-    unsafe {
-        (I2C_MST_ANA_CONF2_REG as *mut u32).write_volatile(
-            // (1 << 18)
-            (I2C_MST_ANA_CONF2_REG as *mut u32).read_volatile() & !I2C_MST_ANA_CONF2,
-        );
-    }
+    modem_lpcon
+        .clk_conf()
+        .modify(|_, w| w.clk_i2c_mst_en().set_bit());
 
     // Before config I2C register, enable corresponding slave.
     match block {
-        REGI2C_BBPLL => {
-            reg_set_bit(I2C_MST_ANA_CONF2_REG, REGI2C_BBPLL_DEVICE_EN);
+        v if v == REGI2C_BBPLL => {
```

Comment:
I wonder if we should have this the preferred style to match constants 🤔 - if yes, we should have that documented somewhere (don't see a fitting document currently for code-style)

But we are also not consistent across the current code base, I think